### PR TITLE
Add the --recover option to the annotate task

### PR DESCRIPTION
**Pull request by EricR86**
This is to address Issue #159.

Forcing a quit-out during annotate and using the recover option seems to work in a spot-check. It's believed this was removed accidentally during the interface change.

Also common options were now moved to a single source so the text relating to the options are no longer duplicated in code.

### CODE REVIEW

**Comment by EricR86** [comment]
@michaelmhoffman this is ready for your review

File: segway/run.py [hunk 1]
```diff
@@ -3944,11 +3944,68 @@ def parse_options(argv):
...
+    class Argument(object):
+        def __init__(self, args, kwargs):
+            self.args = args
+            self.kwargs = kwargs
+    
+    include_coords_args = Argument(
+        args = ["--include-coords"],
+        kwargs = {
+            "metavar": "FILE",
+            "help" : "limit to genomic coordinates in" " FILE (default all)"
```
Comment (michaelmhoffman) [inline segway/run.py:3957]: No space between `"help"` and `:` (this occurs multiple times.

Comment (michaelmhoffman) [reply segway/run.py:3957]: No point to having multiple strings on same line

File: segway/run.py [hunk 2]
```diff
@@ -3944,11 +3944,68 @@ def parse_options(argv):
     identify_finish = tasks.add_parser("", add_help=False)
     identify_init_run = tasks.add_parser("", add_help=False)
 
+    # Common options across tasks
+    class Argument(object):
+        def __init__(self, args, kwargs):
```
Comment (michaelmhoffman) [inline segway/run.py:3949]: I would `def __init__(self, *args, **kwargs):`. Then you can do things like

```
include_coords_args = Argument("--include-coords",
                               metavar="FILE", help="limit to genomic coordinates)
```

where the interface is identical between use here or passing directly to `argparse.ArgumentParser.add_argument()`

**Review by michaelmhoffman — CHANGES_REQUESTED** [review]
Approved after changes. If there's a reason not to do the `Argument.__init__()` like I suggested you can skip it, otherwise I think it would be a useful improvement.

File: segway/run.py (hunk 1 above)
Comment (michaelmhoffman) [reply segway/run.py:3957]: Changing `Argument.__init__()` as suggested would obviate the need for this.

**Pull request merged**